src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationHandler.cs

```diff
@@ -104,50 +108,44 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
 
         protected override string FormatScope() => string.Join(",", Options.Scope);
 
+        private async Task<JsonDocument> CopyPayloadAsync(Dictionary<string, StringValues> content)
```

Comment:
If there's only one parameter ever passed in, you could refactor this method to have the signature `(string propertyName, string value)` and just write them directly into the response, rather than creating a dictionary and iterating it.

Reply:
Has been adjusted